src/components/v2/appDetails/AppDetails.component.tsx

```diff
@@ -23,6 +24,12 @@ const AppDetailsComponent = ({
     const appDetails = IndexStore.getAppDetails();
     const Host = process.env.REACT_APP_ORCHESTRATOR_ROOT;
 
+    useEffect(() => {
+     if( appDetails?.appType === AppType.EXTERNAL_HELM_CHART && params.appId){
```

Comment:
Can you please check whether we want to do it for apps with deploymentAppType -> `helm`

Reply:
No, it is required just in case of externalhelmapp

Reply:
Got it